src/configuration/services/pagespeed.md

```diff
@@ -0,0 +1,41 @@
+# PageSpeed Module
+
+PageSpeed is a family of 4 tools developped by Google Inc to optimize of websites performance.
```

Comment:
The "Inc" is unnecessary.  Or if it is, it needs a comma before it and period after.  (English is weird.)

Reply:
Just unnecessary. Remove :)